source/content/shibboleth-sso.md

```diff
@@ -195,18 +195,24 @@ You can now visit the subdirectory `/simplesaml` on your development site and co
 
 If you are using the [simpleSAMLphp Authentication](https://www.drupal.org/project/simplesamlphp_auth) module, follow the instructions listed in the [README](https://git.drupalcode.org/project/simplesamlphp_auth). These instructions cover both Composer and non-Composer implementations for Drupal sites.
 
-<Alert title="Note" type="info">
+### Composer settings.php File
+
+Configuration of the `setting.php` file is not needed if you are using Composer. 
+
+### Non-Composer settings.php File
 
-If you are using Composer, configuration of the `setting.php` is not needed. For non-Composer implementations, you can add the following lines to `settings.php` so that the Drupal module can locate SimpleSAMLphp:
+Non-Composer implementations must add the following lines to the `settings.php` file to allow the Drupal module to locate SimpleSAMLphp:
 
-**Drupal 7**
+**Drupal 7 Example**
 
 ```php:title=settings.php
 # Provide universal absolute path to the installation.
 $conf['simplesamlphp_auth_installdir'] = $_ENV['HOME'] .'/code/private/simplesamlphp';
 ```
 
-</Alert>
+### Drupal 10
+
+Drupal 10/Symfony 6 is incompatible with the recommended site-level SSO solution for Pantheon Drupal sites outlined in this document. However, the 4.x branch (Dev version) of the Drupal module can use the dev-master version of the SimpleSAMLphp library to achieve Drupal 10 compatibility.
```

Comment:
The simplesamlphp library is not yet compatible with Symfony 6 (the Symfony version included in Drupal 10) except on the dev-master branch. There is a new simplesamlphp_auth branch (4.x) that you could use if you need this solution in Drupal 10.

To use it, require the Drupal module like this:

```
composer require drupal/simplesamlphp_auth:"^4"
```

Remember this means using dev versions so use it at your own risk.



@whitneymeredith what about something like this? ^

Reply:
I like your words a lot more! Thanks!